Dockerfile

```diff
@@ -1,21 +1,45 @@
-FROM node:10.15.3-alpine
+### Stage 0 - Base image
+FROM node:10.15.3-alpine as base
 
 ARG npm_token
 ENV nexus_token=$npm_token
-
 WORKDIR /app
+RUN apk --no-cache update && \
+    apk --no-cache upgrade && \
+    apk add --no-cache --virtual .build-dependencies python make g++ && \
```

Comment:
my understanding of the virtual packages installation was that it can be deleted once it was used for the builds. I do not see any DEL statement anywhere. Do I miss anything?

Reply:
Good catch. 
I 'll add this step in production image (or earlier in base image ? the dependencies won't be available in development image) : 
`RUN apk del .build-dependencies`